e2e-tests/addJoke.test.ts

```diff
@@ -2,10 +2,7 @@ import { chromium, expect, test } from "@playwright/test";
 
 test("Typing a joke and a pun and clicking the seperate buttons should render the text on the homepage", async () => {
   // Simualte a chromium browser with an attribute to slow the running time down by 3 seconds so that you can visually see the test going through each step.
-  const browser = await chromium.launch({
```

Comment:
Thank you for removing slowMo and headless false, the tests work so much better now, running smoother and faster 🧪🤩